<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>酒店主机 —— 前台</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>601</width>
      <height>361</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page_frontdesk">
     <widget class="QPushButton" name="bt_callManager">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>280</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>管理员功能</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_checkout">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>220</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>结账</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_simpleBill">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>220</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>查看账单</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_detailedBill">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>220</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>查看详单</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_roomIdnotValid">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>60</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="palette">
       <palette>
        <active>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>120</red>
            <green>120</green>
            <blue>120</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="text">
       <string>该房间不存在！</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>60</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>房间号：</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="sb_roomId">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>50</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="room_bt_widget" native="true">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>110</y>
        <width>371</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: #ffffff;</string>
      </property>
      <widget class="QRadioButton" name="rb_3">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>10</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #000000;
font-size: 16px;</string>
       </property>
       <property name="text">
        <string>3</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="rb_4">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>10</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #000000;
font-size: 16px;</string>
       </property>
       <property name="text">
        <string>4</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="rb_0">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>10</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #000000;
font-size: 16px;</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="rb_5">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>40</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #000000;
font-size: 16px;</string>
       </property>
       <property name="text">
        <string>5</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="rb_1">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>10</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #000000;
font-size: 16px;</string>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="rb_2">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>10</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #000000;
font-size: 16px;</string>
       </property>
       <property name="text">
        <string>2</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="rb_6">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>40</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #000000;
font-size: 16px;</string>
       </property>
       <property name="text">
        <string>6</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="rb_7">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>40</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #000000;
font-size: 16px;</string>
       </property>
       <property name="text">
        <string>7</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="rb_8">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>40</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #000000;
font-size: 16px;</string>
       </property>
       <property name="text">
        <string>8</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="rb_9">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>40</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #000000;
font-size: 16px;</string>
       </property>
       <property name="text">
        <string>9</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
